package_managers/apt_get/apt_get.bzl

```diff
@@ -77,6 +71,8 @@ def _impl(ctx):
         install_commands = install_commands,
     )
 
+    shell_file_contents.append('#!/bin/bash')
```

Comment:
Did you mean to make these changes?

Reply:
Yes, this was generating malformed bash and breaking a couple of our bazel tests.